src/VkBootstrap.cpp

```diff
@@ -863,10 +911,69 @@ std::vector<std::string> check_device_extension_support(
+	out.vertexPipelineStoresAndAtomics = dest.vertexPipelineStoresAndAtomics || src.vertexPipelineStoresAndAtomics;
+	out.fragmentStoresAndAtomics = dest.fragmentStoresAndAtomics || src.fragmentStoresAndAtomics;
+	out.shaderTessellationAndGeometryPointSize = dest.shaderTessellationAndGeometryPointSize || src.shaderTessellationAndGeometryPointSize;
+	out.shaderImageGatherExtended = dest.shaderImageGatherExtended || src.shaderImageGatherExtended;
+	out.shaderStorageImageExtendedFormats = dest.shaderStorageImageExtendedFormats || src.shaderStorageImageExtendedFormats;
+	out.shaderStorageImageMultisample = dest.shaderStorageImageMultisample || src.shaderStorageImageMultisample;
+	out.shaderStorageImageReadWithoutFormat = dest.shaderStorageImageReadWithoutFormat || src.shaderStorageImageReadWithoutFormat;
+	out.shaderStorageImageWriteWithoutFormat = dest.shaderStorageImageWriteWithoutFormat || src.shaderStorageImageWriteWithoutFormat;
+	out.shaderUniformBufferArrayDynamicIndexing = dest.shaderUniformBufferArrayDynamicIndexing || src.shaderUniformBufferArrayDynamicIndexing;
+	out.shaderSampledImageArrayDynamicIndexing = dest.shaderSampledImageArrayDynamicIndexing || src.shaderSampledImageArrayDynamicIndexing;
+	out.shaderStorageBufferArrayDynamicIndexing = dest.shaderStorageBufferArrayDynamicIndexing || src.shaderStorageBufferArrayDynamicIndexing;
+	out.shaderStorageImageArrayDynamicIndexing = dest.shaderStorageImageArrayDynamicIndexing || src.shaderStorageImageArrayDynamicIndexing;
+	out.shaderClipDistance = dest.shaderClipDistance || src.shaderClipDistance;
+	out.shaderCullDistance = dest.shaderCullDistance || src.shaderCullDistance;
+	out.shaderFloat64 = dest.shaderFloat64 || src.shaderFloat64;
+	out.shaderInt64 = dest.shaderInt64 || src.shaderInt64;
+	out.shaderInt16 = dest.shaderInt16 || src.shaderInt16;
+	out.shaderResourceResidency = dest.shaderResourceResidency || src.shaderResourceResidency;
+	out.shaderResourceMinLod = dest.shaderResourceMinLod || src.shaderResourceMinLod;
+	out.sparseBinding = dest.sparseBinding || src.sparseBinding;
+	out.sparseResidencyBuffer = dest.sparseResidencyBuffer || src.sparseResidencyBuffer;
+	out.sparseResidencyImage2D = dest.sparseResidencyImage2D || src.sparseResidencyImage2D;
+	out.sparseResidencyImage3D = dest.sparseResidencyImage3D || src.sparseResidencyImage3D;
+	out.sparseResidency2Samples = dest.sparseResidency2Samples || src.sparseResidency2Samples;
+	out.sparseResidency4Samples = dest.sparseResidency4Samples || src.sparseResidency4Samples;
+	out.sparseResidency8Samples = dest.sparseResidency8Samples || src.sparseResidency8Samples;
+	out.sparseResidency16Samples = dest.sparseResidency16Samples || src.sparseResidency16Samples;
+	out.sparseResidencyAliased = dest.sparseResidencyAliased || src.sparseResidencyAliased;
+	out.variableMultisampleRate = dest.variableMultisampleRate || src.variableMultisampleRate;
+	out.inheritedQueries = dest.inheritedQueries || src.inheritedQueries;
+}
```

Comment:
This effectively doesn't do anything, since `out` is modified but that is never returned nor copied anywhere. Did you mean to do `dest = out` at the end?

Reply:
Yep, bugs bugs bugs. This PR definitely isn’t fit in its current state (which is fine, it was a bit rushed)

Reply:
Fixed